README.md

```diff
@@ -57,10 +57,87 @@ AirbrakesV2/
 │   ├── ...
 ├── main.py [main file used to run on the rocket]
 ├── constants.py [file for constants used in the project]
-├── pyproject.toml [configuration file for the project]
+├── pyproject.toml [confipip install mkdocs-mermaid2-pluginguration file for the project]
 ├── README.md
 ```
 
+### System Architecture Flowchart
+```mermaid
+%%{init: {'theme': 'dark'}}%%
+flowchart TD
+
+    %% Define styles for different shapes
+    classDef circular fill:#44007e,stroke:#fff,stroke-width:2px,rx:50%,ry:50%;  %% Ovals for classes
+    classDef bubble fill:#164b6c,stroke:#fff,stroke-width:2px,rx:10px,ry:10px; %% Bubbles for methods
+    classDef outputSquare fill:#044728,stroke:#fff,stroke-width:2px; %% Squares for outputs
+    classDef textStyle fill:none,color:#fff,font-weight:bold,font-size:16px; %% Text style for bold white text
+
+    %% Main structure with main.py at the top
+    subgraph mainFlow[Main Flow]
+      direction TB
+      mainpy((main.py)):::circular --> Airbrakes((Airbrakes Context)):::circular
+     
+    end
+    Airbrakes --> Update[update]:::bubble
+    %% IMU Data Packet Flow
+    IMUDataPacket --> Update
+    
+    %% States as individual nodes
+    State((State)):::circular
+    Flightstate((Flight)):::circular
```

Comment:
```suggestion
    CoastState((Flight)):::circular
```
sorry I didn't write the correct state on my diagram. It's called as CoastState now instead of FlightState. Make sure to update the other references too.